src/main/java/org/zalando/planb/revocation/config/FallbackTokenInfoRequestExecutor.java

```diff
@@ -0,0 +1,53 @@
+package org.zalando.planb.revocation.config;
+
+import com.google.common.collect.ImmutableList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.client.RestClientException;
+import org.zalando.stups.oauth2.spring.server.DefaultTokenInfoRequestExecutor;
+import org.zalando.stups.oauth2.spring.server.TokenInfoRequestExecutor;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Executor that allows several tokeninfo endpoints separated by "|", then requests
+ * by order of definition in the string, and if an error happens, falls-back to the next one.
+ *
+ * @author vroldanbetan
+ *
+ */
+public class FallbackTokenInfoRequestExecutor implements TokenInfoRequestExecutor {
+
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    private List<TokenInfoRequestExecutor> executors;
+
+    public FallbackTokenInfoRequestExecutor(final String tokenInfoEndpointUrl) {
+        executors = buildExecutorsFromMultipleURLs(tokenInfoEndpointUrl);
+    }
+
+    private List<TokenInfoRequestExecutor> buildExecutorsFromMultipleURLs(String tokenInfoEndpointUrl) {
+        final List<TokenInfoRequestExecutor> tokenInfoRequestExecutors = new ArrayList<>();
+        for (String endpoint : tokenInfoEndpointUrl.split("\\|")) {
```

Comment:
is a comma not "nicer to read"? (comma will most probably not be part of any sane token info URL)